### ISSUE TITLE
Fudge Cone not animated

### ISSUE THREAD

**Issue by MichaelaRob** [MichaelaRob/frozen-not-in-time#1]
@hellachella 
Hi Chelle, just wondering if you could point me in the right direction. Markbot is coming up green except for one problem saying "The website isn't functioning as expected: The fudge ice cream cone isn't animated" I'm honestly not sure what this even means, would you be able to help me out?

**Comment by hellachella** [comment]
Try using absolute positioning rather than translate(y) and translate(x) to move the icecream.  

```
@keyframes myanimation {
     0% {
           bottom: 100%;
      }
     100% {
          bottom: (a number you figure out)px;
     }
}

**Comment by MichaelaRob** [comment]
@hellachella 
Okay, I just tried this and now the cone doesn't start off the screen anymore 
.cone {
  transform: translate(-40px, 375px);
  position: absolute;
  animation: cone .3s .5s ease-in-out;
  animation-fill-mode: forwards;
}
would I change the coordinates in the transform: translate to make it go off screen?

**Comment by hellachella** [comment]
Remove `transform: translate` and replace with `bottom: 100%;`

**Comment by MichaelaRob** [comment]
yes I did this, but the cone starts in the middle of the screen now
.cone {
  transform: translate(-40px, 375px);
  position: absolute;
  animation: cone .3s .5s ease-in-out;
  animation-fill-mode: forwards;
}

@keyframes cone {

  0% {
    bottom: 100%;
  }

  to {
    bottom: -255px;
  }

}

**Comment by hellachella** [comment]
.cone {
<strike>transform: translate(-40px, 375px);</strike>
position: absolute;
<strong>bottom:100%;
left: (number you must figure out)px;</strong>
animation: cone .3s .5s ease-in-out;
animation-fill-mode: forwards;
}
